file_paths.md

```diff
@@ -280,6 +280,13 @@ These sections describe other Puppet packages that rely on puppet-agent to creat
                 server_datadir            # :server_data
                 yaml                      # :yamldir
 
+    /var/log/puppetlabs                   # :logdir                      /var/lib/puppet/log
+        puppetserver/                     # writeable by puppetserver
+            puppetserver.log
```

Comment:
According to the [original confluence document](https://confluence.puppetlabs.com/pages/viewpage.action?pageId=33948725), this was supposed to be

> Logs: /var/log/puppetlabs/$project.log

And PUP-3809 just said:

> /var/log/puppetlabs <= logfiles go here 

Can we make `/var/log/puppetlabs` owned by `root:puppet` and writable to the group? I assume not because `puppet-agent` creates the directory and there's likely not a `puppet` user?

Will other server-side apps run as `puppet` user? If so, will they have the same issue, e.g. puppetdb?